src/operations/handlers/TestHandler.php

```diff
@@ -37,7 +40,7 @@ public function apply(mixed &$document, object $patch): void
         if (!$this->isJsonEquals($item, $patch->value)) {
             throw new FailedTestException(
                 sprintf(
-                    'Test operation failed asserting that "%s" equals "%s"',
+                    'TestHandler operation failed asserting that "%s" equals "%s"',
```

Comment:
```suggestion
                    'Test operation failed asserting that "%s" equals "%s"',
```
Because the asserting operation is a standard JSON Patch test, it's clearer to report failures as `Test` rather than `TestHandler`, which might suggest an internal error rather than an expected assertion failure

Reply:
Apologies. that change was accidental.  I didn't notice at first that when I renamed the classes PHPStorm had also added `Handler` everyhere the word appeared in a string. I thought I'd reverted all of those (that was part of the later force-push to this branch) but I missed this one. Thanks for catching it.